File: Assignment8/program5.c
////////////////////////////////////////////////////////////////
//
//  File Name   : program5.c
//  Description : Accept a Integer and returns conversion from square feet to square meter
//  Author      : Akhilesh.P.Sonavane.
//  Date        : 17/05/2025
//
////////////////////////////////////////////////////////////////

#include<stdio.h>

////////////////////////////////////////////////////////////////
// 
//  Function Name : SquareMeter
//  Description   : Returns conversion of area square feet to square Meter for input integer
//  Input         : Integer
//  Output        : Double
//
////////////////////////////////////////////////////////////////
double SquareMeter(int iValue)
{
     double Meter = iValue*0.0929;
     return Meter;
}
int main()
{
    int iValue = 0;
    double iRet = 0;
    
    printf("Enter area in square feet");
    scanf("%d",&iValue);
    
    iRet=SquareMeter(iValue);
    
    printf(" squareMeter : %lf",iRet);
    
    return  0;
}

File: Assignment20/program5.java
////////////////////////////////////////////////////////////////
//
//  File Name   : program5.java
//  Description : Accept a N Integer & returns Product of Odd Elements
//  Author      : Akhilesh.P.Sonavane.
//  Date        : 29/05/2025
//
////////////////////////////////////////////////////////////////

import java.util.Scanner;

////////////////////////////////////////////////////////////////
// 
//  Class Name    : Display
//  Function Name : Product
//  Description   : Inputs N Integer & returns Product of Odd Elements 
//  Input         : Integer
//  Output        : Integer
//
////////////////////////////////////////////////////////////////

class Display
{
    public int Product(int[] iPtr, int iSize)
    {
        int iCnt = 0;
        int iCount = 0;

        for(iCnt = 0, iCount = 1; iCnt < iSize; iCnt++)
        {
            if((iPtr[iCnt] % 2) != 0)
            {
                iCount = iCount * iPtr[iCnt];
            }
            else
            {
                iCount = 0;
            }
        }
        return iCount;
    }
}

public class program5 
{
    public static void main(String Args[])
    {
        int[] iArr;
        int iCnt = 0;
        int iLength = 0;
        int iRet = 0;
        
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Number of elements : ");
        iLength = sobj.nextInt();

        iArr = new int[iLength];

        for(iCnt = 0; iCnt < iLength; iCnt++)
        {
            System.out.println("Element "+(iCnt+1)+" : ");
            iArr[iCnt] = sobj.nextInt();
        }

        Display dobj = new Display();

        iRet = dobj.Product(iArr, iLength);

        System.out.println("The Product of Odd Elements : "+iRet);
    }    
}
